ui/app/hooks/useTokenDisplayValue.js

```diff
@@ -0,0 +1,38 @@
+import { getTokenValue, calcTokenAmount } from '../helpers/utils/token-util'
+import { getTokenData } from '../helpers/utils/transactions.util'
+import { useMemo } from 'react'
+
+/**
+ * Defines the shape for the Token input parameter for useTokenDisplayValue
+ * @typedef {Object} Token
+ * @property {string} symbol   - The string to use as a suffix for the token (eg. DAI)
+ * @property {decimals} number - The number of decimals to show when displaying this type of token
```

Comment:
Whoops - looks like these got switched around
```suggestion
 * @property {number} decimals - The number of decimals to show when displaying this type of token
```